diff --git a/configure b/configure
index 70350a07b..135e8b64c 100755
--- a/configure
+++ b/configure
@@ -324,14 +324,6 @@ if [ "${WITH_SPIDERMONKEY}" = "true" ] && [ "${ERLANG_OS}" = "unix" ]; then
         *)  SM_HEADERS="mozjs-${SM_VSN}"
             ;;
         esac
-
-    # This list is taken from src/couch/rebar.config.script, please keep them in sync.
-    if [ ! -d "/usr/include/${SM_HEADERS}" ] && \
-        [ ! -d "/usr/local/include/${SM_HEADERS}" ] && \
-        [ ! -d "/opt/homebrew/include/${SM_HEADERS}" ]; then
-        echo "ERROR: SpiderMonkey ${SM_VSN} is not found. Please specify with --spidermonkey-version."
-        exit 1
-    fi
 fi
 
 # If spidermonkey was disabled but JS_ENGINE set to "spidermonkey", reset it to "quickjs"
